HW1 Crystal Resume Webpage.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+<html lang="en">
+<head>
+<style>
+h1 {
+    font-family: "Calibri";
+	text-align: center;
+}
+
+p {
+    font-family: "Calibri";
+	background-color:#999999;
+	color: #FFFFFF;
+	text-align: center;
+
+}
+body {
+    background-image: url("background.jpg");
+	background-size: contain;
+	background-repeat:no-repeat;
+	width:1024px;
+	height:2000px;
+}
+div.picture1 {
+   width:135px;
+   height:158px;
+   background-image:url("profile.jpg");
+   background-size: contain;
+   background-repeat:no-repeat;
+   background-color: white;
+   border: solid white 4px;
+   float: left;
+
+}
+.container {
+      background-color: #add8e6;
+	  border-color: #999999;
+	  border-style: solid;
+	  border-width: medium;
+      width:30%;
+	  color: #2f4f4f;
+}
+ 
+ .navbar {
+      margin-bottom: 0;
+      border-radius: 0;
+}
+
+footer {
+      background-color: #f2f2f2;
+      padding: 25px;
+    }
+p.fl {
+    font-size: 150%;
+}
+
+
+</style>
+
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+<!-- jQuery library -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+
+<title> Greetings!
+</title>
+
+</head>
+
+<body>
+
+
+<p>
+2181 Elderwood Rd<br>Houston, TX<br>
+832-252-1337 <br>crystal.lee@gmail.com
+
+<div class="container text-center">    
+  <h3>Crystal Lee</h3>
+  <br>
+  <div class="row">
+    <div class="col-sm-2">
+      <img src="profile.jpg" class="img-responsive" style="width:100%" alt="Image">
+    </div>
+    <div class="col-sm-10">
+      <div class="well">
+       <p class="fl">Motivated scientist looking for project management role</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<p class="container"; align="left">
+<strong>WORK EXPERIENCE</strong><br>Aug 2011-Present: R&D Scientist<br> Aug 2007-Aug 2011: Manufacturing Scientist
+</p>
+<p class="container"; align="center">
+<strong>EDUCATION</strong><br>Master of Science in Microbiology<br>Bachelor of Science in Cell and Molecular Biology
+</p>
+<p class="container"; align="right">
+<strong>SKILLS</strong><br>Microbiology Techniques<br>SAP<br>Project Management<br>H2S Awareness<br>Presentation Skills
+</p>
+<p class="container"; align="right">
+<strong>PATENTS/PUBLICATIONS</strong><br>"Green Biocide" Middle East Conference and Exhibition 2012<br>"Onsite DNA Analysis" NACE 2014<br>Patent No. GR-1015687
+</p>
+</body>
+
+<div class="container text-center">    
+  <h3>Interests</h3>
+  <br>
+  <div class="row">
+    <div class="col-sm-4">
+      <img src="cobbler.jpg" class="img-responsive" style="width:100%" alt="Image">
```

Comment:
For images, you can look at [W3schools](https://www.w3schools.com/bootstrap/bootstrap_images.asp) to use some of the "fancy" effects. but it's up to you.